package.json

```diff
@@ -17,7 +17,7 @@
     "prepublishOnly": "npm run build",
     "publish": "lerna publish",
     "publish-preview": "lerna publish from-package",
-    "test": "jest --coverage --verbose",
+    "test": "lerna run --no-bail --ignore e2e-browser test",
```

Comment:
just running `jest --coverage --verbose` like this means that the mocks from browser clobber those from node and vice-versa; Instead we're using lerna to run the all tests in isolation, if one package fails in it's tests, it'll keep going, but report the correct error exit code.